doc/onboarding/view/step1.md

```diff
@@ -0,0 +1,59 @@
+
+## Step 1/5 : Welcome!
+
+### URI
+
+`:userID/welcome`
+
+### Models
+
+<pre>
+user = {
+	&lt;string&gt; name,
+	&lt;id&gt; id,
+	&lt;email&gt; email,
+	&lt;boolean&gt; is_registered: false
+}
+</pre>
+<pre>
+step = {
```

Comment:
My vision was that the step here should be a Backbone `StepModel`, which encapsulate our onboarding `Step` object.

* Backbone `StepModel` because we are in a Backbone/Marionette application and we should keep the most part of its logic
* Our onboarding `Step` object because it allow us to decentralize the business logic and stay framewok-agnostic